module/Application/view/partial/main-nav.phtml

```diff
@@ -173,6 +173,7 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
                             <li><a href="<?= $this->url('activity_calendar') ?>"> <?= $this->translate('Option
                             calendar') ?></a></li>
                         <?php endif; ?>
+                        <li><a href="<?= $this->url('activity/career') ?>"> <?= $this->translate('Career related') ?></a></li>
```

Comment:
Also, the button is currently displayed when on the career page. It should removed there.